fuel-core-interfaces/src/block_importer.rs

```diff
@@ -1,4 +1,9 @@
+use super::model::{ConsensusVote, FuelBlock, SealedFuelBlock};
+use fuel_types::Bytes32;
+use std::sync::Arc;
+
 /// Currently just placeholder for new block included and new block created events.
+/// TODO remove this after relayer pull request passes
```

Comment:
Not sure what this todo is referring to. New blocks come either from the synchronizer or peers during bft, not the relayer.